docs/cap_anndata_schema.md

```diff
@@ -1266,13 +1266,13 @@ Key-value pair in the `uns` dictionary
 	</tr>
 	<tr>
   		<td><b>example</b></td>
-  		<td><code>'['John Smith', 'Cody Miller', 'Sarah Jones']'</code></td>
+  		<td><code>'[{'author_name':'John Smith', 'email':'jsmith@university.edu', 'orcid' :'ORCID:0000-0002-3843-3472'}]'</code></td>
```

Comment:
It is not clear what to do in case of empty fields. Should we still write keys with empty values like `'email': None` or we can just skip it.

Reply:
We should write keys with empty values @rm1113